File: demos/python/drop_in.py
from openai import OpenAI
import llamanet
llamanet.run()
client = OpenAI()
stream = client.chat.completions.create(
  model="gpt-3.5-turbo",
  messages=[{"role": "user", "content": "What is the meaning of life?"}],
  stream=True,
)
for chunk in stream:
  if chunk.choices[0].delta.content is not None:
    print(chunk.choices[0].delta.content, end="")

File: demos/js/drop_in.js
// Demonstrate the instant drop-in replacement in action
const OpenAI = require('openai')
const llamanet = require("../../index");
const msg = `what comes after 1 2 3 5 8? give me as many variations as possible with reasons`;
async function main() {
  await llamanet.run()
  const openai = new OpenAI()

  // even thought the model says 'gpt-4o', since running in llamanet, the request starts the default model first
  const stream = await openai.chat.completions.create({
    model: "gpt-4o",
    messages: [{ role: 'user', content: msg }],
    stream: true,
  });
  for await (const chunk of stream) {
    process.stdout.write(chunk.choices[0]?.delta?.content || '');
  }
}
main();
